### PR TITLE
Add information on README

### PR DESCRIPTION
**Pull request by mateusortiz**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,18 @@
-# harmonic-website
-Harmonic website
+![Harmonic Website](images/thumb.png)
+
+Harmomic is built using the mordern JavaScript (ES2015/ES6 - ES2016/ES7). Take a look at the code, start learning the new JavaScript and fell free to contribute.
```
Comment (UltCombo) [inline README.md]: fell -> feel

Comment (UltCombo) [inline README.md]: Maybe: ES2015/ES6 - ES2016/ES7 -> ES2015/ES6+

Comment (UltCombo) [inline README.md]: TODO: reflect these changes in the site content.

Comment (mateusortiz) [inline README.md]: reflect changes in the theme of harmonic :+1:

**Comment by UltCombo** [comment]
Awesome, thanks! :smile:

File: README.md (hunk 1 above)
Comment (UltCombo) [inline README.md]: https://github.com/JSRocksHQ/harmonic-theme-selfblog/issues/25

**Pull request merged**